Sources/Public/BacktraceBreadcrumbsLogManager.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+
+@objc public class BacktraceBreadcrumbsLogManager: NSObject {
+    
+    private var breadcrumbId = Date().millisecondsSince1970
```

Comment:
Probably call this `lastBreadcrumbId` and auto increment when using it

Reply:
The releavance of lastBreadcrumbId is different, lastBreadcrumbId is used to bind it in report with key "breadcrumbs.lastId", So I think current change is good enough. As per other comment we made increment id irrespective of result of add breadcrumb.